README.md

```diff
@@ -68,8 +68,18 @@ GC_THREADS=9
   ```
   
     Note: Don't leave any hyphen/special characters for db_name.
-
-6. There are 2 types of tpcds benchmark script provided,
+	
+4. To verify the TPCDS database creation and row count of tables with expected standard counts for DB. Use the script provided in the utils directory.
+    It can be used if you have chosen DB size - 1GB/1TB/2TB/3TB/10TB/30TB/100TB.
+  
+  ```
+   get_tpcds_count.sh <db_name>
```

Comment:
Accept DB sizes like 10GB/10gb.. rather than db_name as we don't follow file naming conversions.

Reply:
Changes made to take dbsize also argument and using same to store the standard numbers.
Dbname will be required to run queries for existing  DB, so keeping it one of param.